src/assets.rs

```diff
@@ -226,33 +222,71 @@ impl<'a> DerefMut for AssetEditor<'a> {
 
 /// Assets stored in an assets store.
 pub struct Assets {
+    /// A key-value store for assets.
     pub store: AssetsStore,
+
+    /// A set of asset codes loaded from verified asset libraries.
+    pub verified_assets: HashSet<AssetCode>,
```

Comment:
Both these fields should be private (the whole point of this class is that you only access it through the CRUD operations defined below)